### ISSUE TITLE
:banana: Gorillas :banana:

### ISSUE THREAD

**Issue by KrulPax** [ScorpioStation/ScorpioStation#339]
:banana:
https://github.com/tgstation/tgstation/pull/29845
https://github.com/tgstation/tgstation/pull/29923
https://github.com/tgstation/tgstation/pull/30113
https://github.com/tgstation/tgstation/pull/30878
:banana:
This series of PRs introduce the following things:
- A chance for monkeys to turn into hostile gorillas under exposure to radiation (most commonly at the genetics lab)
- A geneticist/CMO traitor item: a serum that allows the player to turn monkeys and people into gorillas (sentient ones as well) through injection
- gibbed/butchered gorillas will drop brains for brain extraction

:banana:
I am sending this here as more of a talking point since because of the differences between our servers this idea might not be as fun as it is at TG, especially with low pop to consider, however, I am confident that those PRs would make geneticist job a little more interesting and spicy, especially as a traitor. The question is, is this kind of spice welcome here, or would it be too overwhelming? I'd like to invite y'all to discuss this here with me.
:banana::banana::banana::banana::banana:

**Comment by blinkdog** [comment]
Will the hostile gorillas spawn with any musical instruments?
https://www.youtube.com/watch?v=1V_xRb0x9aw

Seriously though, I think this would be a fun feature.

**Comment by Kiyahitayika** [comment]
@blinkdog Since this changes Genetics and DNA, I object until the DNA refactor is done.

I am also not a complete fan of how the code is implemented:

```
	if(radiation)
		if(radiation > 100)
			if(!IsKnockdown())
				emote("collapse")
			Knockdown(200)
			to_chat(src, "<span class='danger'>You feel weak.</span>")

		if(radiation > 30 && prob((radiation - 30) * (radiation - 30) * 0.002))
			gorillize()
			return
			
```
This has a chance to happen ANY time you're changing ANY Genetic block. This may be confusing to folks who irradiate a block and have a Gorilla spawn. They might think, "oh no, that is the Gorilla block" when in fact there is no Gorilla block.

**Comment by Kiyahitayika** [comment]
> A geneticist/CMO traitor item: a serum that allows the player to turn monkeys and people into gorillas (sentient ones as well) through injection

Not all species have 'deform' forms. This will _gib_ species, such as Vox.

**Comment by KrulPax** [comment]
> I think I'd be in favor of this spice if we weren't so low pop. Something to definitely revisit at another time. I really appreciate this request. The instagibbing isn't so much the fault of this code, but issues with the species' codes. Apologies if I come on hard in this comment; I'm definitely a fan of adding new things and this looks pretty fun. If others can put my concerns at ease, please go ahead!

I have to agree with this in the end - while the "deform" forms issue could be handled by making gorilla its own thing, the low pop would result in creating some chaos. Well, I invite y'all to consider this, once we will have an environment that wouldn't make this idea too OP
